### PR TITLE
Thickener & Dewatering Unit Model Documentation

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Summary/Motivation:
Adds documentation for the thickener and dewatering unit models, both of which are based off the IDAES separator

## Changes proposed in this PR:
- Adds unit model documentation
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/unit_models/dewatering.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Introduction
+------------
+Dewatering is a process commonly used in wastewater treatment plants to separate water from solids using centrifugal motion
+and/or vacuums. In doing so, the dewatering unit reduces the sludge volume, which improves system operation and mitigates costs
+related to storage, processing, disposal, etc. This implementation of the dewatering unit is based on the `IDAES separator unit <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/separator.html>`_.
+
+Degrees of Freedom
+------------------
+The degrees of freedom in a dewatering unit model are the the inlet feed state variables:
```
Comment (agarciadiego) [inline docs/technical_reference/unit_models/dewatering.rst]:
```suggestion
The degrees of freedom in a dewatering unit model are the inlet feed state variables:
```

File: docs/technical_reference/unit_models/thickener.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Introduction
+------------
+Sludge thickening is a process commonly used in wastewater treatment plants to increase the sludge concentration of a stream
+as well as reduce its volume by removing free water. In doing so, the thickener reduces the load of downstream processes,
+such as digestion and dewatering. This implementation of the thickener unit is based on the `IDAES separator unit <https://idaes-pse.readthedocs.io/en/stable/reference_guides/model_libraries/generic/unit_models/separator.html>`_.
+
+Degrees of Freedom
+------------------
+The degrees of freedom in a thickener unit model are the the inlet feed state variables:
```
Comment (agarciadiego) [inline docs/technical_reference/unit_models/thickener.rst]:
```suggestion
The degrees of freedom in a thickener unit model are the inlet feed state variables:
```

File: docs/technical_reference/unit_models/dewatering.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Suspended solid concentration", ":math:`C_{TSS} = 0.75 (C_{X_{I}} + C_{X_{IP}} + C_{X_{BH}} + C_{X_{BA}} + C_{X_{S}})`"
+   "Dewatering factor", ":math:`f_{dewat} = p_{dewat} (\frac{10}{C_{TSS}})`"
+   "Remove factor", ":math:`f_{q_{du}} = \frac{TSS_{rem}}{100 * f_{dewat}}`"
+   "Overflow particulate fraction", ":math:`split_{particulate} = 1 - TSS_{rem}`"
+   "Overflow soluble fraction", ":math:`split_{soluble} = 1 - f_{q_{du}}`"
+
+Classes
```
Comment (adam-a-a) [inline docs/technical_reference/unit_models/dewatering.rst]:
```suggestion
Class Documentation
```

File: docs/technical_reference/unit_models/dewatering.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Phases", ":math:`p`", "['Liq']"
+   "Particulate Components", ":math:`j`", "['X_I', 'X_S', 'X_P', 'X_BH', 'X_BA', 'X_ND']"
+   "Non-particulate Components", ":math:`j`", "['H2O', 'S_I', 'S_S', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'S_ALK']"
```
Comment (adam-a-a) [inline docs/technical_reference/unit_models/dewatering.rst:41]: I think it'd be nice to refer the reader to ASM for people to follow the component symbols

**Review by agarciadiego — APPROVED** [review]

**Review by adam-a-a — APPROVED** [review]

**Review by luohezhiming — APPROVED** [review]
Looks good to me. I find that both units are only applicable to ASM1, that means once we move to ASM2d system, we need to consider to upgrate these two units, and it will need more work and documentation updates.

**Review by luohezhiming — APPROVED** [review]
Looks good to me. I find that both units are only applicable to ASM1, that means once we move to ASM2d system, we need to consider to upgrate these two units, and it will need more work and documentation updates.

**Review by luohezhiming — APPROVED** [review]
Looks good to me. I find that both units are only applicable to ASM1, that means once we move to ASM2d system, we need to consider to upgrate these two units, and it will need more work and documentation updates.

**Pull request merged**